go.mod

```diff
@@ -47,13 +52,16 @@ require (
 	github.com/ory/dockertest/v3 v3.6.2
 	github.com/oschwald/geoip2-golang v1.4.0
 	github.com/phayes/freeport v0.0.0-20180830031419-95f893ade6f2
+	github.com/pkg/errors v0.9.1
 	github.com/rs/cors v1.7.0
 	github.com/spf13/cobra v1.1.1
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.7.1
+	github.com/sqs/goreturns v0.0.0-20181028201513-538ac6014518 // indirect
 	github.com/stretchr/objx v0.2.0 // indirect
 	github.com/stretchr/testify v1.6.1
 	github.com/textileio/go-ds-mongo v0.1.2
+	github.com/textileio/go-threads v1.0.2
```

Comment:
Not sure what's going on here. Didn't think Powergate uses threads at all...

Reply:
huh... i was going to use Threads for results collection but found that file on disk worked better at this point. will go back to it when i do a Hub version of this w/ buckets. maybe it got stuck in there though...

Reply:
Prob just need to run `go mod tidy`